cluster/manifests/coredns-local/configmap-local.yaml

```diff
@@ -6,6 +6,43 @@ metadata:
   labels:
     application: coredns
 data:
+  unbound.conf: |
+    server:
+      directory: "/opt/unbound/etc/unbound/"
+      interface: 0.0.0.0
+      interface-automatic: yes
+      # Drop user privileges after binding the port.
+      username: "_unbound"
+      # Prevent the unbound server from forking into the background as a daemon
+      do-daemonize: no
+      # log to stderr
+      use-syslog: no
+      log-servfail: yes
+      # allow query localhost (coredns at 127.0.0.1:9254)
+      do-not-query-localhost: no
+      access-control: 0.0.0.0/0 allow
+      harden-dnssec-stripped: no
+      so-reuseport: yes
+      cache-min-ttl: 1
+      disable-dnssec-lame-check: yes
+      minimal-responses: yes
+      extended-statistics: yes
+      # support reverse lookup of kubernetes addresses
+      local-zone: "2.10.in-addr.arpa." transparent
+      local-zone: "3.10.in-addr.arpa." transparent
+      # make metrics available for the unbound-telemetry container (127.0.0.1:8953)
```

Comment:
8953 -> 9054 as far as I read the pod spec

Reply:
Thanks, updated the comment!